site/content/blog/2019-10-16-debugging-using-charles.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Debugging Using Charles"
+slug: debugging-using-charles
+date: 2019-10-16T00:00:00-04:00
+author: Jesse Hallam
+github: lieut-data
+community: jesse.hallam
+---
+
+I recently acquired a copy of [Charles](https://www.charlesproxy.com), the well known Web Debugging Proxy Application. I've actually stumbled across this product on multiple occasions, but never bothered to actually try it... almost exclusively because I thought the website looked a little dated. In trying to suss out the root cause behind [MM-19091](https://mattermost.atlassian.net/browse/MM-19091), I needed a way to debug against our [community](https://community.mattermost.com) servers but running with my local copy of the [mattermost webapp](https://github.com/mattermost/mattermost-webapp). This would allow me to insert `console.log` debugging statements and even use the [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) without the minified component names.
```

Comment:
```suggestion
I recently acquired a copy of [Charles](https://www.charlesproxy.com), the well-known Web Debugging Proxy Application. I've actually stumbled across this product on multiple occasions, but never bothered to actually try it... almost exclusively because I thought the website looked a little dated. In trying to suss out the root cause behind [MM-19091](https://mattermost.atlassian.net/browse/MM-19091), I needed a way to debug against our [community](https://community.mattermost.com) servers but running with my local copy of the [mattermost webapp](https://github.com/mattermost/mattermost-webapp). This would allow me to insert `console.log` debugging statements and even use the [React Developer Tools](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en) without the minified component names.
```

Reply:
Just changed 'well known' to 'well-known'